methods_and_flow_control/string_practice.rb

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+
+practice_string = 'this is a string to practice with'
+
+puts practice_string
+puts practice_string.capitalize
+puts practice_string.upcase
+puts practice_string.capitalize.sub('string', "'string'")
+puts "The string '#{practice_string}' has #{practice_string.length} characters"
+puts practice_string.reverse
+puts 'practice practice practice!'
```

Comment:
Can you find a method you can call on `practice_string` to accomplish this output?